package lab14_Exceptions;

public class Q5 {
    public static void main(String[] args) {
        throw new RuntimeException("Own Message");
    }
}
